### PR TITLE
add message_to_high_level_string debug helper

### PR DESCRIPTION
**Pull request by rukai**
Allows quickly debugging including high level message representations in debug logs when debugging transforms.

For example now we can just write:
```rust
tracing::error!("messages: {}", message_wrapper.messages_to_high_level_string());
```

Instead of:
```rust
tracing::error!(
    "messages: {}",
    message_wrapper.
            .messages
            .iter_mut()
            .map(|x| x.to_high_level_string())
            .collect::<Vec<_>>()
);
```

Starting the method name with `messages_` means that the method will be suggested when the user writes: 
`tracing::error!("messages: {}", message_wrapper.messages`

### CODE REVIEW

File: shotover-proxy/src/transforms/mod.rs [hunk 1]
```diff
@@ -512,6 +512,15 @@ impl<'a> Wrapper<'a> {
         }
     }
 
+    pub fn messages_to_high_level_string(&mut self) -> String {
```
Comment (conorbros) [inline shotover-proxy/src/transforms/mod.rs:515]: Can we put `#[cfg(test)]` on it? Make it clear you shouldn't be using this outside of a debugging/test context.

**Review by conorbros — APPROVED** [review]
Great idea. LGTM

File: shotover-proxy/src/transforms/mod.rs (hunk 1 above)
Comment (rukai) [reply shotover-proxy/src/transforms/mod.rs:517]: good idea.
done!

**Comment by rukai** [comment]
Oh wait, cant use #[test] because that can only be used in actual test modules.

**Comment by rukai** [comment]
I swapped it to use `#[cfg(feature = "alpha-transforms")]` instead of `#[cfg(test)]` 
`#[cfg(test)]` is no good because its inaccessible from within shotover during an integration test.
but alpha-transforms is a reasonable way to ensure we dont accidentally check in debug prints that use this

**Review by benbromhead — APPROVED** [review]

**Pull request merged**